httpfilecontext.h

```diff
@@ -49,6 +50,8 @@ class HTTPFileContext : public IHTTPContext
 	long connectTimeout;
 	long followLocation;
 	long timeout;
+	long maxSendSpeed;
+	long maxRecvSpeed;
```

Comment:
According to the cURL documentation these should be of type `curl_off_t`.

Reply:
yeap, i will change it later, curl_off_t is int64_t I think.